src/SIL.LCModel/Application/ApplicationServices/XmlImportData.cs

```diff
@@ -1005,7 +1002,7 @@ private void ReadXmlObject(XmlReader xrdr, FieldInfo fi, ICmObject objToUse)
 					else
 						cmo = factory.Create(guid, (IPhPhonemeSet)fi.Owner);
 					// Remove the default code added by PhTerminalUnit.SetDefaultValuesAfterInit in OverridesLing_Lex.
-					(cmo as PhBdryMarker).CodesOS.Clear();
+					((PhBdryMarker)cmo).CodesOS.Clear();
```

Comment:
there's another place in this method where it does a similar pattern, but instead of casting it does this:
```suggestion
					(cmo as PhBdryMarker)?.CodesOS.Clear();
```